README.md

```diff
@@ -65,3 +65,52 @@ noirup --path ./git/noir
 **Tip**: All flags have a single character shorthand equivalent! You can use `-v` instead of `--version`, etc.
 
 ---
+
+## Github Action
+
+This action is in early development and so will likely experience breaking changes. It's recommended to pin to a particular commit.
+
```

Comment:
I'm happy to add release-please and versioning to noirup to help people pin stuff

Reply:
Yeah, this is something I wanted to have a little think about. That's one potential option but we could also go down the route of https://github.com/dtolnay/rust-toolchain where the action's version represents the toolchain version to be installed.

The trouble with that is that if breaking changes to noirup itself are made then you could get hit with that despite pinning to a particular tag.

Reply:
I think the branch versions of `rust-toolchain` is a little too clever because you can't use it with variables (so you can't build a matrix). I'd rather have a separately versioned action that take input for the toolchain I want.

Reply:
Yeah, I'm leaning the same way, Between that, no way to version changes in noirup and potential annoyances in enumerating all noir versions, we're giving up too much for a slightly more concise CI file.

Reply:
I'm going to have a go at setting up release please on this repo.